docs/en/administration/Configuration.md

```diff
@@ -674,6 +674,105 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Introduced in**: v3.2.0
+
+##### lake_publish_version_max_threads
+
+- **Default**: 128
+- **Description**: The maximum number of threads for Version Publish tasks.
+- **Introduced in**: v3.2.0
+
+##### lake_autovacuum_parallel_partitions
+
+- **Default**: 8
+- **Description**: The maximum number of partitions that can undergo AutoVacuum simultaneously. AutoVaccum is the Garbage Collection after Compactions.
+- **Introduced in**: v3.2.0
+
+##### lake_autovacuum_partition_naptime_seconds
+
+- **Unit**: Seconds
+- **Default**: 180
+- **Description**: The minimum interval between AutoVacuum operations on the same partition.
+- **Introduced in**: v3.2.0
+
+##### lake_autovacuum_grace_period_minutes
+
+- **Unit**: Minutes
+- **Default**: 5
+- **Description**: The time range for retaining historical data versions. Historical data versions after Compactions within this time range are not automatically cleaned via AutoVacuum. You need to set this value greater than the maximum query time to avoid that the data accessed by running queries get deleted before the queries finish.
+- **Introduced in**: v3.2.0
+
+##### lake_autovacuum_stale_partition_threshold
+
+- **Unit**: Hours
+- **Default**: 12
+- **Description**: If a partition has no updates (loading, DELETE, or Compactions) within this time range, the system will not perform AutoVacuum on this partition.
+- **Introduced in**: v3.2.0
+
+##### lake_enable_ingest_slowdown
+
+- **Default**: false
+- **Description**: Whether to enable Data Ingestion Slowdown. When Data Ingestion Slowdown is enabled, if the Compaction Score of a partition exceeds `lake_ingest_slowdown_threshold`, loading tasks on that partition will be throttled down.
+- **Introduced in**: v3.2.0
+
+##### lake_ingest_slowdown_threshold
+
+- **Default**: 100
+- **Description**: The Compaction Score threshold that triggers Data Ingestion Slowdown. This configuration only takes effect when `lake_enable_ingest_slowdown` is set to `true`.
+- **Introduced in**: v3.2.0
+
+> **Note**
+>
+> When `lake_ingest_slowdown_threshold` is less than `lake_compaction_score_selector_min_score`, the effective threshold will be `lake_compaction_score_selector_min_score`.
+
+##### lake_ingest_slowdown_ratio
+
+- **Default**: 0.1
+- **Description**: The ratio of the loading rate slowdown when Data Ingestion Slowdown is triggered.
+- **Introduced in**: v3.2.0
+
+Data loading tasks consist of two phases: data writing and data committing (COMMIT). Data Ingestion Slowdown is achieved by delaying data committing. The slowdown ratio is calculated with the following formula: `(compaction_score - lake_ingest_slowdown_threshold) * lake_ingest_slowdown_ratio`. For example, if the data writing phase takes 5 minutes, `lake_ingest_slowdown_ratio` is 0.1, and the Compaction Score is 10 higher than `lake_ingest_slowdown_threshold`, the delay in data committing time is `5 * 10 * 0.1 = 5` minutes, which is equivalent to doubling the average loading speed.
```

Comment:
1. The slowdown ratio is calculated --- not clear, see question in chinese    2. calculated using the following formula   3. which is equivalent to doubling the average loading speed. --- "doubling" is not correct, which means the average loading speed is halved.